.github/workflows/ci.yml

```diff
@@ -43,6 +43,12 @@ jobs:
         toolchain:
           - stable
           - nightly
+        partition:
+          - 1/5
+          - 2/5
+          - 3/5
+          - 4/5
+          - 5/5
```

Comment:
I think 4 would be enough, WDYT?

Reply:
Sure, 5 was just an arbitrary number to be honest